test/core/commands/codeBlockExpectedResult.txt

```diff
@@ -1 +1 @@
-{"Code":"A = B;\nC=1;", "LineIndices": [0, 1],"InPorts": ["B"], "OutPorts": ["A","C"], "Data": "null"}
\ No newline at end of file
+{"Code":"A = B;\nC=1;", "LineIndices": [0, 1],"InPorts": ["B"], "OutPorts": ["A","C"]}
```

Comment:
This is the right result of the test, so it looks good. Let it fail for now and after another PR gets in, it should be passing.